src/Compilers/CSharp/Test/Semantic/Semantics/RefExtensionMethodsTests.cs

```diff
@@ -38,6 +38,136 @@ public static void Main()
                 Diagnostic(ErrorCode.ERR_RefLvalueExpected, "5").WithLocation(13, 9));
         }
 
+        [Fact]
```

Comment:
>Fact [](start = 9, length = 4)

nit: consider adding `WorkItem(25862, "[https://github.com/dotnet/roslyn/issues/25862](https://github.com/dotnet/roslyn/issues/25862)")` on the added tests